### PR TITLE
WIP for initials models.

### PR DESCRIPTION
**Pull request by rquijanol**
Added 

ShoppingList
ShoppingListItem
Ingredient
MeasuringUnit

All of these models relay on primarykeys created by Django and  have Timestamp fields

### CODE REVIEW

File: easyshopping/settings.py [hunk 1]
```diff
@@ -37,9 +37,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'easyshoppinglist'
 ]
 
-MIDDLEWARE = [
+MIDDLEWARE_CLASSES = [
```
Comment (LeenaBhegade) [inline easyshopping/settings.py]: what version of Django did you use ? The version I used is Django==1.10.7 , I think its better to use latest version. For Django version 1.10 and above we should be keeping MIDDLEWARE as is

File: easyshoppinglist/models.py [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+    # missing link to user
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class ShoppingListItem(models.Model):
+    shopping_list = models.ForeignKey(ShoppingList, on_delete=models.CASCADE)
+    ingredient = models.ForeignKey(Ingredient)
+    quantity = models.DecimalField(decimal_places=2, max_digits=10)
+    created_at = models.DateTimeField(auto_now_add=True)
```
Comment (LeenaBhegade) [inline easyshoppinglist/models.py]: can we have size field which indicates size of the ingredient, for eg small potato , medium onion

Comment (rquijanol) [inline easyshoppinglist/models.py]: Ok, I'll add the choices to the model

File: easyshopping/settings.py (hunk 1 above)
Comment (rquijanol) [inline easyshopping/settings.py]: Yes, its something to do with my environment. I have the std mac osx python but also have 3.5 installed.

I'll fix that

**Comment by LeenaBhegade** [comment]
For environment you can try this.May be install virtual wrapper and create
virtual environment and after that do "workon" environment name. Pip
install django~=1.10.0

**Pull request merged**